data/microos/butane/script

```diff
@@ -59,20 +87,9 @@ ExecStart=/usr/bin/touch /var/log/flagfile
 WantedBy=multi-user.target
 EOF
 
-cat << EOF > /etc/systemd/system/home.mount
-[Unit]
-Before=local-fs.target
-Requires=systemd-fsck@dev-disk-by\x2dpartlabel-testing_part.service
-After=systemd-fsck@dev-disk-by\x2dpartlabel-testing_part.service
-
-[Mount]
-Where=/home
-What=/dev/disk/by-partlabel/testing_part
-Type=ext4
-
-[Install]
-RequiredBy=local-fs.target
-EOF
+if test -e /dev/disk/by-partlabel/testing_part; then
+    systemd_mount
```

Comment:
Shouldn't this be directly after create_fs? Then the `test -e` is also not necessary.